e2e_tests/tests/cluster/test_exp_continue.py

```diff
@@ -0,0 +1,318 @@
+import tempfile
+from typing import List, Tuple
+
+import pytest
+
+from determined.common import yaml
+from determined.common.api import bindings
+from determined.common.api.bindings import experimentv1State
```

Comment:
I suspect this line and line 13 are two lines which violate I2041. changing this will allow not excluding it in `.flake8`.